controller.rst

```diff
@@ -64,9 +34,9 @@ class::
     class LuckyController
     {
         /**
-         * @Route("/lucky/number/{max}")
+         * @Route("/lucky/number/{max}", name="app_lucky_number")
```

Comment:
Is this `app_` prefix a good practice for route names? I'd never do that and it makes routes a bit long ... but if you consider it a good practice, then it's OK.